tasks.go

```diff
@@ -135,6 +138,10 @@ func GetNagiosTasks(tasks chan<- Task, projects []string, basepath string) {
 			tasks <- GetNagiosHistoricalData(p, pod, outFor, errOutFor)
 		}
 	}
+
+	if !foundANagiosPod {
+		fmt.Println("We could not find a Nagios pod in any project, which will reduce the usefulness of this tool, for a full analysis please ensure that Nagios is running in all RHMAP projects.")
```

Comment:
Looks good.
Some minor rephrasing might make it read better though
e.g. "A Nagios pod could not be found in any project. For a more thorough analysis, please ensure Nagios is running in all RHMAP projects"